commons-hpcc/src/main/java/org/hpccsystems/commons/errors/HpccErrorLevel.java

```diff
@@ -0,0 +1,32 @@
+package org.hpccsystems.commons.errors;
+
+/**
+ * The severity of an error.
+ */
+public enum HpccErrorLevel
+{
+    FATAL, ERROR, WARNING, INFO;
+
+    public static HpccErrorLevel getErrorLevel(int eclenum)
+    {
+        if (eclenum == 0)
+        {
+            return null;

```

Comment:
what does null represent in this case? no error? unknown error level?
also wha about eclenum < 0 ?

Reply:
The error code in the soap-generated ecl error objects is an int, so 0=null.